release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -366,6 +366,11 @@ For more information, refer to the following sources:
 * Installer-provisioned infrastructure: xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#modifying-install-config-for-dual-stack-network_ipi-install-installation-workflow[Deploying with dual-stack networking]
 * User-provisioned infastructure: xref:../installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc#installation-configuration-parameters-network_installing-bare-metal-network-customizations[Network configuration parameters]
 
+[id="ocp-4-13-nodeselector-egressfirewall-enhancement"]
+==== Node selector added to egress firewall for OVN-Kubernetes network plugin
+
+In {product-title} {product-version}, `nodeSelector` has been added to OVN-Kubernetes network plugin. Users can now specify a label applied to one or more nodes and the node IP of selected nodes will be included in that rule. For more information, see xref:../networking/ovn_kubernetes_network_provider/configuring-egress-firewall-ovn.adoc#configuringNodeSelector-example_configuring-egress-firewall-ovn[Example nodeSelector for EgressFirewall]
```

Comment:
>Users can now specify a label applied to one or more nodes and the node IP of selected nodes will be included in that rule. 

When I first read this, I read _specify a label applied to one or more nodes and _specify a label applied to_ the node IP of selected nodes _

Maybe something like: Users can now specify a label applied to one or more nodes, which causes the node IP of selected nodes to be included in that rule. (Does that make sense?)